app/views/roles/_new_modal.html.erb

```diff
@@ -13,25 +13,29 @@
 				<div class='field'>
 					<div class='field-inline'>
 						<input type="radio" name="role[name]" value="nonprofit_admin" id='roleName--admin'>
-						<label for='roleName--admin'><strong>Nonprofit Admin</strong></label>
+						<label for='roleName--admin'><strong><%= t('roles.new.nonprofit_admin') %></strong></label>
 					</div>
-					<p>Full permissions, including bank payouts</p>
+					<p><%= t('roles.new.full_permissions_including_bank_payouts') %></p>
 				</div>
 
 				<div class='field'>
 					<div class='field-inline'>
 						<input type="radio" name="role[name]" value="nonprofit_associate" checked='checked' id='roleName--assoc'>
-						<label for='roleName--assoc'><strong>Nonprofit Associate</strong></label>
+						<label for='roleName--assoc'><strong><%= t('roles.new.nonprofit_associate') %></strong></label>
 					</div>
-					<p>Edit content and view records</p>
+					<p><%= t('roles.new.edit_content_and_view_records') %></p>
 				</div>
 
-				<p><strong>Important!<br/> Double check this email address</strong></p>
-				<input name='role[email]' type='email' placeholder='Email' id='role-email-field' required><br>
-				<input name='role[email_confirmation]' type='email' placeholder='Confirm email' id='role-email-field' required parsley-equalto='#role-email-field'>
+				<p><strong>
+					<%= t('roles.new.important') %>
+					<br>
+					<%= t('roles.new.double_check_this_email_address') %>
+				</strong></p>
+				<input name='role[email]' type='email' placeholder="<%= t('roles.new.email') %>" id='role-email-field' required><br>
+				<input name='role[email_confirmation]' type='email' placeholder="<%= t('roles.new.confirm_email') %>" id='role-email-field' required parsley-equalto='#role-email-field'>
 
 				<p class='error'></p>
-				<%= render 'common/modal_submit', loading: 'Creating...', text: 'Create' %>
+				<%= render 'common/modal_submit', loading: t('roles.new.creating'), text: t('roles.new.create') %>
```

Comment:
One minor tweak: "Create" is such a common UI text element that it might make sense to add it to ui.en.yml and refer to it from there. 

Thoughts?

Reply:
Makes sense!

Reply:
I think it applies "Creating..." too